corehq/apps/README.rst

```diff
@@ -0,0 +1,203 @@
+   in the development domain and then pushed to the production domain, where real project data is entered.
+   Linked domains are also used by certain enterprise-type projects that manage one program across multiple regions
+   and use a separate downstream domain for each region.
+registration
+   Workflows for creating new accounts on HQ.
+reminders
+   A subset of SMS/messaging, including functionality around incoming SMS keywords. "Reminders" is a leftover term from a previous iteration of the messaging framework.
+saved_reports
+   Functionality to let users save a specific set of report filters and optionally run reports with those filters on a scheduled basis.
+toggle_ui
+   Framework for feature flags, which are used to limit internal feature to specific domains and/or users.
+translations
+   Functionality for managing application translations, including integration with Transifex, which is used by a small number of projects.
+user_importer
+   Bulk importing of users.
+
+Tertiary Apps
+^^^^^^^^^^^^^
+These apps may be useful parts of the system but don't have as much active development as the groups above.
+
+aggregate_ucrs
+   An experimental framework for creating more complex reporting pipelines based off the UCR framework.
+analytics
+   Integrations with third-party analytics tools such as Google Analytics and Kissmetrics.
+   Also contains internal product-focused tools such as AB testing functionality.
+builds
+   Models relating to CommCare Mobile builds, so that app builders can control which mobile version their apps use.
+   Some of this app relates to J2ME builds: historically, CommCare mobile supported both J2ME and Android devices.
+   The J2ME functionality is largely deprecated, but most of the related code remains, in both this app and ``app_manager``.
+case_search
+   Models and utils related to searching for cases using Elasticsearch. Used for Case Claim and the Case List Explorer. 
+dashboard
+   The tiled UI that acts as the main landing page for HQ.
+formplayer_api
+   Functionality interacting with formplayer, primarily used by SMS surveys.
+mobile_auth
+   Generates the XML needed to authorize mobile users.
+notifications
+   "Banner" notifications used by the support team to notify users of upcoming downtime,
+   ongoing issues, etc.
+receiverwrapper
+   Contains the API for receiving XML form submissions. This app mostly deals with the interfacing portion of the
+   API including auth, rate limiting etc. but not the actual data processing which is contained in the
+   `form_processor` app.
+settings
+   API keys and 2FA functionality.
+smsbillables
+   Billing functionality relating to charging for SMS, allowing us to pass carrier charges on to clients.
+smsforms
+   SMS surveys, a part of messaging that allow end users to interact with a CommCare form via SMS instead of
+   via mobile or Web Apps.
```

Comment:
```suggestion
   SMS surveys allow end users to interact with a CommCare form via SMS instead of
   via mobile or Web Apps. This is part of ``messaging``.
```